spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteDataFilesSparkAction.java

```diff
@@ -276,29 +215,25 @@ RewriteDataFilesCommitManager commitManager(long startingSnapshotId) {
   }
 
   private Builder doExecute(
-      RewriteExecutionContext ctx,
-      Stream<RewriteFileGroup> groupStream,
-      RewriteDataFilesCommitManager commitManager) {
+      RewritePlanResult planResult, RewriteDataFilesCommitManager commitManager) {
     ExecutorService rewriteService = rewriteService();
 
     ConcurrentLinkedQueue<RewriteFileGroup> rewrittenGroups = Queues.newConcurrentLinkedQueue();
 
     Tasks.Builder<RewriteFileGroup> rewriteTaskBuilder =
-        Tasks.foreach(groupStream)
+        Tasks.foreach(planResult.groups())
             .executeWith(rewriteService)
             .stopOnFailure()
             .noRetry()
             .onFailure(
-                (fileGroup, exception) -> {
-                  LOG.warn(
-                      "Failure during rewrite process for group {}", fileGroup.info(), exception);
-                });
+                (fileGroup, exception) ->
+                    LOG.warn(
```

Comment:
Not sure why this was changed? Did Spotless get more strict?

Reply:
My mistake. I changed the line in my previous version, and fixed an IntelliJ warning too, then reverted my change but not the removal of the curly brackets

Reply:
reverted